cypress.config.js

```diff
@@ -0,0 +1,9 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'https://discuss.flarum.org/', 
+    setupNodeEvents(on, config) {
+    },
+  },
```

Comment:
I don't see any usage of the setupNodeEvents function.
Usually, if a function is not used, it should be removed from the codebase.